backend/lambdas/api/authorizer/tests/test_cognito.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+
+from joserfc import jwk
+import responses
```

Comment:
Is responses in our pipfile?

Reply:
Good catch -- we're actually using responses in a bunch of other tests, but looks like it never got added to the Pipfile.

Reply:
We'll explicitly add responses to the Pipfile as part of the next round of dependency updates.